### PR TITLE
Multiple changes to disentangle epi and endothelial cells

### LINKED ISSUE #2417: [Class hierarchy] remove classification of endothelial cell under epithelial cell
**CL term**

endothelial cell

**Suggested revision of class hierarchy**

Remove classification of endothelial cells as epithelial.  While there may be some abstract conceptual justification, the resulting classifications are just too confusing for most users.  e.g. no one working on lungs would expect this.

image

Note - this will require more work that just removing a single assertion as there are multiple manual assertions to remove.  This is in part because at some point in the history of CL - inferences where instantiated.

e.g. 

image

### PR DESCRIPTION
**Pull request by dosumis**
Some sidetrack into work on fenestrated endothelial cells. 
Still to do - various kidney cells - where epi- vs endothelium may become ambiguous (?)

Fixes #2417

### CODE REVIEW

**Comment by dosumis** [comment]
#gogoeditdiff

**Comment by dosumis** [comment]
#gogoeditdiff

**Comment by dosumis** [comment]
Update to pato import needed.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -22603,6 +22602,16 @@ SubClassOf(obo:CL_0008039 obo:CL_0000100)
 SubClassOf(obo:CL_0008039 ObjectSomeValuesFrom(obo:RO_0002120 obo:CL_0008002))
 SubClassOf(obo:CL_0008039 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_7742))
 
+# Class: obo:CL_0008040 (squamous endothelial cell of venule)
+
+AnnotationAssertion(Annotation(rdfs:comment "GOC:tfm") obo:IAO_0000115 obo:CL_0008040 "A endothelial cell of the venule that is squamous shaped. This is in contrast to the cubodial shape of high endothelial venule cells.")
+AnnotationAssertion(dc:contributor obo:CL_0008040 "https://orcid.org/0000-0002-7073-9172")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:22608]: The ORCID is a literal. Please, add in the IRI editor.

**Review by aleixpuigb — COMMENTED** [review]
Please, see inline comments

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -26999,9 +27007,6 @@ SubClassOf(obo:CL_1000742 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0070252))
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1000746 "KUPO:0001036")
 AnnotationAssertion(rdfs:label obo:CL_1000746 "glomerular cell")
 EquivalentClasses(obo:CL_1000746 ObjectIntersectionOf(obo:CL_1000612 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000074)))
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply src/ontology/cl-edit.owl:27009]: It's created as a post-process, it's not added to the edit file.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (anitacaron) [reply src/ontology/cl-edit.owl:22608]: That's why the QC is failing.

Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:22608]: Can we remove annotation properties that we don't use anymore? It creates confusion.

File: src/ontology/cl-edit.owl (hunk 2 above)
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:27009]: Is it added to the -simple file?

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -4167,8 +4169,7 @@ AnnotationAssertion(owl:deprecated obo:CL_0000070 "true"^^xsd:boolean)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") obo:IAO_0000115 obo:CL_0000071 "An endothelial cell that lines the vasculature.")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000071 "cuboidal endothelial cell of vascular tree")
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:4170]: You mean the synonym?  

Not new in this edit, but will fix.

[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:4170]: Fixed

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -22603,6 +22602,16 @@ SubClassOf(obo:CL_0008039 obo:CL_0000100)
 SubClassOf(obo:CL_0008039 ObjectSomeValuesFrom(obo:RO_0002120 obo:CL_0008002))
 SubClassOf(obo:CL_0008039 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_7742))
 
+# Class: obo:CL_0008040 (squamous endothelial cell of venule)
+
+AnnotationAssertion(Annotation(rdfs:comment "GOC:tfm") obo:IAO_0000115 obo:CL_0008040 "A endothelial cell of the venule that is squamous shaped. This is in contrast to the cubodial shape of high endothelial venule cells.")
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:22607]: FIxed

**Comment by dosumis** [comment]
Seems that cl-edit file is broken.  I'm not sure how that can happen when saving from Protege!  Will roll back edits, but this might need to wait until tomorrow.

File: src/ontology/cl-edit.owl [hunk 5]
```diff
@@ -43,7 +43,7 @@ Annotation(terms:contributor <https://orcid.org/0000-0002-8688-6599>)
 Annotation(terms:contributor <https://orcid.org/0000-0002-9900-7880>)
 Annotation(terms:contributor <https://orcid.org/0000-0003-1980-3228>)
 Annotation(terms:license <http://creativecommons.org/licenses/by/4.0/>)
-Annotation(rdfs:comment "See PMID:15693950, PMID:12799354, PMID:20123131, PMID:21208450; Contact Alexander Diehl, addiehl@buffalo.edu, University at Buffalo.")
+Annotation(rdfs:comment "See PMID:15693950, PMID:12799354, PMID:20123131, PMID:21208450; Contact Alexander Diehl, addiehl"@buffalo.edu, University at Buffalo.)
```
Comment (anitacaron) [inline src/ontology/cl-edit.owl:46]: @dosumis here's the change that broke the owl 😬

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (anitacaron) [reply src/ontology/cl-edit.owl:22608]: I'm not sure, we're importing OMO, which is where it comes from. It's important to make sure you use the correct annotation, and the QC is there to help you.

File: src/ontology/cl-edit.owl (hunk 5 above)
Comment (dosumis) [reply src/ontology/cl-edit.owl:46]: Thanks.  Kinda weird.  I didn't edit that!

**Comment by dosumis** [comment]
#gogoeditdiff

**Comment by dosumis** [comment]
#gogoeditdiff

**Comment by dosumis** [comment]
@aleixpuigb I think now good.  Could you take a look through inference in the reasoned simple file to see if anything jumps out as obviously wrong?

**Comment by dosumis** [comment]
Fixes #1492

**Review by aleixpuigb — APPROVED** [review]
It is probably out of the scope of this PR, but I have detected one inconsistency in `vasa recta cell`, which is a 'kidney capillary endothelial cell' and a 'vein endothelial cell' at the same time. I will open a new ticket to solve that.
All the rest looks good to me.

**Comment by emquardokus** [comment]
> It is probably out of the scope of this PR, but I have detected one inconsistency in `vasa recta cell`, which is a 'kidney capillary endothelial cell' and a 'vein endothelial cell' at the same time. I will open a new ticket to solve that. All the rest looks good to me.

@aleixpuigb vasa recta is both venous and arterial blood. It’s a place where the circulatory system meets. This is likely correct because of that.

**Comment by aleixpuigb** [comment]
> > It is probably out of the scope of this PR, but I have detected one inconsistency in `vasa recta cell`, which is a 'kidney capillary endothelial cell' and a 'vein endothelial cell' at the same time. I will open a new ticket to solve that. All the rest looks good to me.
> 
> @aleixpuigb vasa recta is both venous and arterial blood. It’s a place where the circulatory system meets. This is likely correct because of that.

Yes, the problem is that not all cells are 'capillary endothelial cell' and 'vein endothelial cell'. We should probably have a disjoint so this doesn't happen. I might add that in ticket #2425. For now, I will solve it adding SubclassOf 'microvascular endothelial cell'.

**Pull request merged**